vertical-pod-autoscaler/recommender/model/types.go

```diff
@@ -50,6 +50,11 @@ func CoresFromCPUAmount(cpuAmunt ResourceAmount) float64 {
 	return float64(cpuAmunt) / 1000.0
 }
 
+// QuantityFromCPUAmount converts CPU cores to a resource.Quantity.
```

Comment:
Cores? I tihnk millicores, no?

Reply:
Done. Changed to "ResourceAmount". It shouldn't mention units at all.